system/netdata.service.in

```diff
@@ -43,7 +43,7 @@ CapabilityBoundingSet=CAP_DAC_OVERRIDE
 # is required for apps plugin
 CapabilityBoundingSet=CAP_DAC_READ_SEARCH
 # is required for freeipmi plugin
-CapabilityBoundingSet=CAP_FOWNER
+CapabilityBoundingSet=CAP_FOWNER CAP_SYS_RAWIO
```

Comment:
Have we confirmed that the plugin actually still needs CAP_FOWNER as well as CAP_SYS_RAWIO?

Reply:
I can't confirm nor deny anything myself (have no access to a server with IPMI). This PR is based on users' experience with freeipmi plugin:

- That capability was added in #9234.
- [It was confirmed](https://github.com/NixOS/nixpkgs/pull/87867#issuecomment-632620393) that the capability is needed.
- [It was confirmed](https://github.com/netdata/netdata/issues/10044#issuecomment-1156445663) that sometimes it is not enough and we need CAP_SYS_RAWIO.